arwen/context/arwen.go

```diff
@@ -974,7 +990,9 @@ func (host *vmContext) createETHCallInput() []byte {
 	}
 
 	for _, arg := range host.vmInput.Arguments {
-		newInput = append(newInput, arg...)
+		paddedArg := make([]byte, arwen.HashLen)
```

Comment:
maybe add a new constant to EthereumArgumentLen

Reply:
Called it `ArgumentLenEth`, to be consistent with the other constants in `helpers.go`.